tests/h/indexer_test.py

```diff
@@ -135,6 +123,7 @@ def delete_annotation(self, patch):
 @pytest.fixture
 def celery(patch):
     cel = patch('h.indexer.celery')
-    cel.request = DummyRequest(db=mock.Mock(), es=mock.Mock(), feature=mock.Mock())
+    cel.request = DummyRequest(
+        db=mock.Mock(), es=mock.Mock(), new_es=mock.Mock(), feature=mock.Mock())
```

Comment:
Yes, the other tasks still use `request.es` because they never run when the feature is off (i.e. [here](https://github.com/hypothesis/h/blob/reindex-cli-command/tests/h/indexer_test.py#L28))